src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -75,10 +79,12 @@ g.test('mininum_buffer_binding_size')
     const { type, minBufferBindingSize } = t.params;
```

Comment:
there's a typo in the title of this test ("mininum"); could you fix it since you're editing this file already?

Reply:
Done